packages/cli/core/src/commands/atomic/init.ts

```diff
@@ -0,0 +1,82 @@
+import {CLICommand} from '@coveo/cli-commons/command/cliCommand';
+import {Config} from '@coveo/cli-commons/config/config';
+import {Before} from '@coveo/cli-commons/decorators/before';
+import {Beta} from '@coveo/cli-commons/decorators/beta';
+import {UnknownError} from '@coveo/cli-commons/errors/unknownError';
+import {Flags} from '@oclif/core';
+import inquirer from 'inquirer';
+import {
+  atomicAppPreconditions,
+  atomicLibPreconditions,
+  createAtomicApp,
+  createAtomicLib,
+} from '../../lib/atomic/createAtomicProject';
+
+export default class AtomicInit extends CLICommand {
+  public static description =
+    'Scaffold a project to work with Coveo Atomic Framework';
+
+  public static examples = [
+    '<%= config.bin %> <%= command.id %> --type=app myAwesomeSearchPage',
```

Comment:
Is this the new format for examples in ocliff ?

Reply:
Yeah, if you do oclif generate command, what it spew out. I tested, it works nicely and replace the bin by coveo, and the id by the command or its alias depending on the use case.